pyccel/parser/semantic.py

```diff
@@ -1423,6 +1424,38 @@ def _assign_GeneratorComprehension(self, lhs_name, expr, **settings):
         expr_new.set_fst(expr.fst)
         return expr_new
 
+    def _handle_unamed_var(self, code_block, scope):
+        program_body = [line for line in code_block.body]
+        body = [item for item in program_body]
+        new_code_block = False
+        for ind, line in enumerate(body):
+                if isinstance(line, FunctionCall) or (isinstance(line, Assign) and isinstance(line.rhs, FunctionCall)):
```

Comment:
The `_additional_exprs` variable should allow you to call this function directly from `_visit_FunctionCall`